package_linter.py

```diff
@@ -1974,6 +1974,16 @@ def resource_consistency(self):
                         "When using a postgresql database, you should add postgresql in apt dependencies."
                     )
 
+        if (
+            isinstance(self.manifest["resources"].get('permissions', {}).get('main', {}).get('url'), str)
+            and "init_main_permission" not in self.manifest['install']
+            and not isinstance(self.manifest["resources"].get('permissions', {}).get('main', {}).get('allowed'), str)
```

Comment:
cf the 3rd check ... do we really have situations where it's okay to not ask the admin about which users to allow (and simply hardcode the value in manifest) ?

Imho for webapp main permission, it should *always* be asked to the admin what people to allow ?

Reply:
Dendrite, and I guess Conduit too... these are all apps whose main permission does not point where users usually go. ;)

In the case of the Matrix servers, it's because all three are pure API, but the default endpoint `/_matrix` always returns a 404 error, which is not great for the CI. So we trick it with something like this: https://github.com/YunoHost-Apps/dendrite_ynh/blob/22fc954be5859176a1f67c6e83065e929519f40c/conf/nginx.conf#L4